modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectAuthenticationHandlerImpl.java

```diff
@@ -201,6 +187,35 @@ public void requestAuthentication(
 		}
 	}
 
+	private long _createOpenIdConnectSession(
```

Comment:
@stian-sigvartsen 
Who are the "all impls" in
`OpenIdConnectSession creation/deletion should be managed externally to OpenIdConnectSessionManager. So the model can be used for all impls.`?

It does not need to be used for anyone directly